dids/src/main/kotlin/web5/sdk/dids/methods/key/DidKey.kt

```diff
@@ -144,15 +136,15 @@ public class DidKey(uri: String, keyManager: KeyManager) : Did(uri, keyManager)
 
       val publicKeyJwk = keyGenerator.bytesToPublicKey(publicKeyBytes)
 
-      val verificationMethodId = "${parsedDidUri.uri}#$id"
+      val verificationMethodId = "${parsedDid.uri}#$id"
       val verificationMethod = VerificationMethod.Builder()
         .id(verificationMethodId)
         .publicKeyJwk(publicKeyJwk)
         .controller(did)
         .type("JsonWebKey2020")
```

Comment:
if we're using `JsonWebKey` with `did:jwk` we should use it here too

Reply:
oh yea, this should also be updated! changed to `JsonWebKey`